src/domain/ledger/index.ts

```diff
@@ -4,8 +4,8 @@ export * from "./errors"
 
 export const liabilitiesMainAccount = "Liabilities"
 
-export const toLiabilitiesAccountId = (walletId: WalletId): LiabilitiesAccountId =>
-  `${liabilitiesMainAccount}:${walletId}` as LiabilitiesAccountId
+export const toLiabilitiesWalletId = (walletId: WalletId): LiabilitiesWalletId =>
```

Comment:
I'm confused about this change. The ledger doesn't have a wallet does it? The ledger has accounts or journals or journal entries or something like that but not a wallet or? I don't think wallet is the correct naming for a grouping identifier in the ledger subdomain.
There is clearly some kind of mapping between wallets and the scope of journals in the ledger but I wouldn't use the same naming.

Reply:
so we have, at the domain level:
User
Account
Wallet

That is map at the mongodb level:
user.id
user.id
wallet.id (and it used to be used.id before this PR)

Ledger tx are being identify by wallet.id now.
Ledger still use an internal id, but the domain is not making use of that any more.